docs/src/main/doc/IntegratingDdfSpatial.adoc

```diff
@@ -0,0 +1,2089 @@
+= Integrating {branding} Spatial App
+Copyright (c) Codice Foundation
+2.7-SNAPSHOT
```

Comment:
Should be 2.7.0-SNAPSHOT to be specific....is there any way to get this dynamically? Otherwise I could see this being a pain come release time to change every since file with a new version.